home.css

```diff
@@ -0,0 +1,690 @@
+  border-bottom: 1px solid #ec5242;
+  background-color: #f7f7f8;
+  font-size: 1.2rem;
+  padding-inline: 11%;
+  padding-block: 5%;
+}
+
+.fa-bars {
+  font-size: 2rem;
+}
+
+.first-section {
+  background-image: url(assets/first-section.png);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.first-section-container {
+  width: 92.5%;
+  padding-top: 7rem;
+  margin-inline: auto;
+  min-height: 326.85px;
+  height: auto;
+}
+
+.open-intro-heading {
+  margin-top: 5%;
+  font-size: 1.6rem;
+  font-weight: 400;
+  color: #ec5242;
+}
+
+.open-heading {
+  font-family: 'cocogoose', sans-serif;
+  font-size: 3rem;
+  color: #ec5242;
+  margin: 0;
+}
+
+.first-section-para {
+  padding: 3%;
+  background-color: #f7f7f8;
+  text-align: center;
+  border: 3px solid white;
+  margin-top: 5%;
+}
+
+h3 {
+  font-family: 'cocogoose', sans-serif;
+  font-size: 1.84rem;
+  color: #504f4f;
+}
+
+.venue {
+  font-size: 1.12rem;
+}
+
+/* second-section styling */
+
+.second-section {
+  background-image: url(assets/repeat-bg.png);
+  background-repeat: repeat;
+  background-position: center;
+
+  /* margin: 0; */
+}
+
+.program-list {
+  width: 95%;
+  margin-inline: auto;
+  margin-top: 10%;
+}
+
+.second-section-heading {
+  color: #fff;
+  text-align: center;
+  padding-top: 8%;
+  font-size: 2rem;
+  margin-bottom: 0;
+}
+
+hr {
+  width: 10%;
+  margin-inline: auto;
+  border: none;
+  height: 3px;
+  background: #ec5242;
+}
+
+.flex-box {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  background-color: #393c43b3;
+  margin-block: 2.5%;
+  color: white;
+}
+
+.flex-box img {
+  height: 20%;
+  width: 10%;
+  margin-left: 4%;
+}
+
+.flex-box h2 {
+  font-size: 1.2rem;
+  margin-left: 3%;
+  align-items: center;
+  width: 40%;
+  color: #ec5242;
+  text-align: left;
+  align-self: center;
+}
+
+.flex-box p {
+  font-size: 0.9rem;
+  margin-left: 5%;
+}
+
+.button-container {
+  width: 50%;
+  margin-inline: auto;
+  height: 70px;
+  margin-block: 5%;
+  padding-block: 7%;
+}
+
+.program-button {
+  width: 100%;
+  height: 50px;
+  background-color: #ec5242;
+  border-radius: 4px;
+  border-color: transparent;
+  color: #fff;
+  font-size: 1.1rem;
+  cursor: pointer;
+}
+
+.program-button:hover {
+  background-color: #fff;
+  color: #ec5242;
+  height: 60px;
+  font-weight: 700;
+  width: 110%;
+  font-size: 1.2rem;
+  border-color: #ec5242;
+}
+
+.desktop-container-link {
+  display: none;
+}
+
+/* third section -dynamic one */
+
+.section-speakers-heading {
+  text-align: center;
+}
+
+.featured-speakers {
+  width: 92.5%;
+  margin: auto;
+}
+
+.speaker-section {
+  width: 100%;
+  margin-inline: auto;
+  display: flex;
+  justify-content: space-between;
+  margin-block: 12%;
+}
+
+.speaker-image-section {
+  display: flex;
+  justify-content: center;
+}
+
+.speaker-image {
+  width: 128px;
+  height: 128px;
+
+  /* align-self: center;
+  justify-self: center;
+  margin-block: auto;
+  z-index: 2; */
```

Comment:
- [ ] You should also delete this commented block of code and on lines `570 and 626`